cpp/include/cudf/types.hpp

```diff
@@ -156,7 +156,8 @@ enum type_id {
   TIMESTAMP_MILLISECONDS,  ///< duration of milliseconds since Unix Epoch in int64
   TIMESTAMP_MICROSECONDS,  ///< duration of microseconds since Unix Epoch in int64
   TIMESTAMP_NANOSECONDS,  ///< duration of nanoseconds since Unix Epoch in int64
-  CATEGORY,   ///< Categorial/Dictionary type
+  CATEGORY,   ///< Categorial (type to be removed)
```

Comment:
```suggestion
  CATEGORY,   ///< Categorical (type to be removed)
```

Reply:
Can we remove it now?

Reply:
This is still be referenced by some code. I thought it would be best remove it in a separate PR..